### PR TITLE
Simplify arithmetic operation in logical lines checker

### PR DESCRIPTION
**Pull request by augustelalande**
## Summary

Simplify arithmetic operation in logical lines checker

### CODE REVIEW

**Review by zanieb — APPROVED** [review]
Huh. It'd be good to get the eyes of the original author on this too.

**Review by charliermarsh — APPROVED** [review]

**Comment by charliermarsh** [comment]
Yeah that's interesting, hmm... Maybe it evolved over time? Maybe this was just the way they thought about the logic?

File: crates/ruff_linter/src/checkers/logical_lines.rs [hunk 1]
```diff
@@ -24,7 +24,7 @@ pub(crate) fn expand_indent(line: &str, indent_width: IndentWidth) -> usize {
     let tab_size = indent_width.as_usize();
     for c in line.bytes() {
         match c {
-            b'\t' => indent = (indent / tab_size) * tab_size + tab_size,
```
Comment (charliermarsh) [inline crates/ruff_linter/src/checkers/logical_lines.rs:27]: Wait sorry, _is_ this the same? What if `indent` is 4 and `tab_size` is 3?

`indent += tab_size` would yield 7, but `indent = (indent / tab_size) * tab_size + tab_size` would yield 6, right?

Comment (charliermarsh) [reply crates/ruff_linter/src/checkers/logical_lines.rs:27]: I.e., is the goal here perhaps that `indent` ends up as a multiple of `tab_size`?

Comment (zanieb) [reply crates/ruff_linter/src/checkers/logical_lines.rs:27]: Is the deal here that there's implicit integer rounding?

Comment (zanieb) [reply crates/ruff_linter/src/checkers/logical_lines.rs:27]: If so, can we add a comment explaining the deal? @charliermarsh

**Pull request merged**